examples/basic/basic.ipynb

```diff
@@ -585,7 +585,7 @@
    },
    "outputs": [],
    "source": [
-    "job = batch_client.get_batch_features(\n",
+    "job = batch_client.get_historical_features(\n",
```

Comment:
`By calling the get_batch_features method we are able to retrieve a job object. This object can be used to retrieve the resulting training dataset that is exported by Feast.`

This portion can be updated as well.